unity-renderer/Assets/DCLPlugins/ECS7/ECS7Plugin.cs

```diff
@@ -0,0 +1,24 @@
+using DCL;
+using DCL.ECSRuntime;
+
+public class ECS7Plugin : IPlugin
+{
+    private readonly ComponentCrdtWriteSystem crdtWriteSystem;
+    private readonly IECSComponentWriter componentWriter;
+    private readonly ECS7ComponentsPlugin componentsPlugin;
+
+    public ECS7Plugin()
+    {
+        crdtWriteSystem = new ComponentCrdtWriteSystem(Environment.i.platform.updateEventHandler, Environment.i.world.state);
+        componentWriter = new ECSComponentWriter(crdtWriteSystem.WriteMessage);
+
+        componentsPlugin = new ECS7ComponentsPlugin(DataStore.i.ecs7.componentsFactory, componentWriter);
```

Comment:
The `componentFactory` isn't needed by anybody else (unless I'm mistaken). I would just pass it through a constructor here or even better just make ECS7Plugin to instantiate it (if possible).

Reply:
the factory is used by another system that is not inside this plugin.
I'm waiting for be able to do a benchmark to see if I can move that system inside this plugin

Reply:
I would still avoid using `DataStore.i` and use Dependency Injection, it helps with the tests. I leave it up to you!